CNAME

```diff
@@ -1 +1 @@
-gci16.fossasia.org
```

Comment:
I'd suggest you to read more about `CNAME`.
You arent supposed to change it :)

Links:
* https://help.github.com/articles/using-a-custom-domain-with-github-pages/
* https://en.wikipedia.org/wiki/CNAME_record